src/App.tsx

```diff
@@ -2,6 +2,8 @@ import React, { Fragment, FunctionComponent, useEffect, useState } from 'react';
 import { connect, Provider } from 'react-redux';
 
 import { BrowserRouter as Router, RouteComponentProps, withRouter } from 'react-router-dom';
+import { Slide, ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.minimal.css';
```

Comment:
waarom importeren we deze hier, maar de andere style files in App.scss

Reply:
Klopt had eerst deze import en pas daarna de App.scss aangemaakt, ik zal deze er ook bij zetten.